frontend/apps/web/components/jobs/Form/DestinationOptionsForm.tsx

```diff
@@ -18,6 +20,7 @@ interface DestinationOptionsProps {
   hideInitTableSchema?: boolean;
   hideDynamoDbTableMappings?: boolean;
   destinationDetailsRecord: Record<string, DestinationDetails>;
+  errors?: FieldErrors<DestinationOptionsFormValues>;
```

Comment:
Only sub component that is using this right now is AwsS3, but this sets us up nicely to update the other destination forms to not directly depend on react-hook-form.